restservice/src/main/java/org/itevents/service/geocoder/AddressLocationService.java

```diff
@@ -0,0 +1,12 @@
+package org.itevents.service.geocoder;
+
+import org.itevents.dao.model.Location;
+
+/**
+ * Created by zavodnyuk on 03.04.2016.
+ */
+public interface AddressLocationService {
```

Comment:
@zavodnyuk I think that `GeoCoderService` is more appropriate name for this interface.